### PR TITLE
Addressing misspellings and Gogland warnings.

### PR DESCRIPTION
**Pull request by limbalrings**
# What Are We Doing Here

There were a few misspellings and warnings my IDE identified. Just cleaning those issues up.

## How to Verify

1. Check out this PR
2. Run Command X, Click Button Y
3. Profit

### CODE REVIEW

File: crwatcher/watcher_test.go [hunk 1]
```diff
@@ -19,17 +19,20 @@ import (
...
-	"github.com/wpengine/lostromos/printctlr"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/util/wait"
 	restclient "k8s.io/client-go/rest"
 )
 
+type TestController struct {
+	ResourceController
+}
+
```
Comment (limbalrings) [inline crwatcher/watcher_test.go]: This is done to prevent a circular dependency issue.

File: printctlr/controller.go [hunk 2]
```diff
@@ -17,14 +17,17 @@ package printctlr
...
 )
 
 // Controller provides a crwatcher.ResourceController that prints the events out
 // as the are received. It is a basic implementation that can be used for
-// debuging. It also serves as an example for how you could implement your own
+// debugging. It also serves as an example for how you could implement your own
 // crwatcher.ResourceController.
-type Controller struct{}
+type Controller struct {
+	crwatcher.ResourceController
```
Comment (limbalrings) [inline printctlr/controller.go]: declaring Controller implements this interface

File: test-data/cr_nemo_update.yml [hunk 3]
```diff
@@ -1,11 +0,0 @@
...
-
-apiVersion: stable.nicolerenee.io/v1
-kind: Character
-metadata:
-  name: nemo
-spec:
-  Name: Nemo
-  From: Finding Nemo
-  By: Disney
-  For: Reasons
```
Comment (limbalrings) [inline test-data/cr_nemo_update.yml]: This is removed on make clean, so it should probably not be checked in to the codebase.

File: tmplctlr/controller.go [hunk 4]
```diff
@@ -27,6 +28,7 @@ import (
 // Controller implements a valid crwatcher.ResourceController that will manage
 // resources in kubernetes based on the provided template files.
 type Controller struct {
+	crwatcher.ResourceController
```
Comment (limbalrings) [inline tmplctlr/controller.go]: declaring Controller implements this interface

File: tmplctlr/controller_test.go [hunk 5]
```diff
@@ -64,11 +64,13 @@ func createTestDir(files []templateFile) string {
...
-		if err != nil {
-			log.Fatal(err)
-		}
+		func() {
+			defer f.Close()
+			_, err = io.WriteString(f, file.contents)
+			if err != nil {
+				log.Fatal(err)
+			}
+		}()
```
Comment (limbalrings) [inline tmplctlr/controller_test.go]: This addresses a warning about using defer in a loop. Forcing defer to happen at the end of each anonymous function call

File: test-data/cr_nemo_update.yml (hunk 3 above)
Comment (nicolerenee) [reply test-data/cr_nemo_update.yml]: This file is required. It should not be removed by `make clean`.

**Comment by nicolerenee** [comment]
This code is not currently signed off on and can not be merged.

File: printctlr/controller.go (hunk 2 above)
Comment (nicolerenee) [reply printctlr/controller.go]: This doesn't declare that this is an implementation of crwatcher.ResourceController.
https://golang.org/doc/faq#implements_interface has more details about how go doesn't have "implements" declarations. In that guide it also provides a way to make sure we implement the interface if we wanted to do that we could but this in and of itself is not something that is standard in Go or that actually does anything.

**Pull request merged**